composer.json

```diff
@@ -25,8 +25,7 @@
 			"includes/"
 		],
         "files": [
-            "includes/sensei-functions.php",
-            "includes/3rd-party/3rd-party.php"
```

Comment:
I'm moving the 3rd-party tweaks to be loaded a bit later because they're calling WP functions that are not present that early in a test environment.